testing/test_pluginmanager.py

```diff
@@ -313,6 +313,8 @@ def test_preparse_args(self, pytestpm):
         assert '"hello123"' in excinfo.value.args[0]
         pytestpm.consider_preparse(["-pno:hello123"])
 
+        pytestpm.consider_preparse(["-p"])
```

Comment:
I suggest to add a comment in the lines "just make sure single -p doesn't blow up"